codec-http/src/main/java/io/netty/handler/codec/http/DefaultLastHttpContent.java

```diff
@@ -107,32 +108,24 @@ private void appendHeaders(StringBuilder buf) {
     }
 
     private static final class TrailingHttpHeaders extends DefaultHttpHeaders {
-        private static final class TrailingHttpHeadersNameConverter extends HttpHeadersNameConverter {
-            TrailingHttpHeadersNameConverter(boolean validate) {
-                super(validate);
-            }
+        private static final class TrailingHttpHeadersNameValidator implements
+                                                                    DefaultHeaders.NameValidator<CharSequence> {
+
+            private static final TrailingHttpHeadersNameValidator INSTANCE = new TrailingHttpHeadersNameValidator();
```

Comment:
nit: Accessing a private field of an enclosing class will produce synthetic accessor method. Please make it package-private for the inner beauty. :-)